src/sentry/integrations/slack/sdk_client.py

```diff
@@ -0,0 +1,24 @@
+from slack_sdk import WebClient
+
+from sentry.models.integrations import Integration
+from sentry.services.hybrid_cloud.integration import integration_service
+from sentry.silo.base import SiloMode
+
+
+class SlackClient(WebClient):
```

Comment:
we have a function call `track_response_data` in all of our in-house ApiClients, i'm wondering how we might do the same for this SlackClient... duplicate the method signatures, call super(), and then track the response?